tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/PetResource.java

```diff
@@ -412,15 +427,18 @@ public Response findPetsByTags(
     public Response updatePetWithForm (
         @Parameter(
             name = "petId",
+            in = ParameterIn.PATH,
             description = "ID of pet that needs to be updated",
             required = true)
         @PathParam("petId") Long petId,
         @Parameter(
             name = "name",
+            in = ParameterIn.QUERY,
```

Comment:
Since this method argument is annotated with `@FormParam` I'm not sure this is right.